tag.sh

```diff
@@ -15,7 +15,8 @@ for dir in ${VERSIONS}; do
   IMAGE_ID=$(cat .image-id)
   name=$(docker inspect -f "{{.Config.Labels.name}}" $IMAGE_ID)
   version=$(docker inspect -f "{{.Config.Labels.version}}" $IMAGE_ID)
-  date_and_hash="$(date +%Y%m%d)-$(git rev-parse --short HEAD)"
+  commit_date=$(git show HEAD --format=%cd --date=short | sed 's/-//g')
```

Comment:
to not have the diff in $commit_date, I need to add --quiet (at least git 2.20.1)

Reply:
Ah, I was doing it on a diff-less commit, hence I didn't catch it :)

I can't find a `--quiet` option for `git show`, but it appears `-s` is the appropriate flag. Amending.

Reply:
looks fine now, thanks